src/normal_form_game.jl

```diff
@@ -499,6 +552,39 @@ Base.summary(g::NormalFormGame) =
            " ",
            split(string(typeof(g)), ".")[end])
 
+# delete_action
+
+"""
+    delete_action(g, action[, player_idx=1])
+
+Return a new `NormalFormGame` instance with the action(s) specified by `action`
+deleted from the action set of the player specified by `player_idx`.
+
+# Arguments
+
+- `g::NormalFormGame` : `NormalFormGame` instance.
+- `action::Union{PureAction, AbstractVector{<:PureAction}}` : The action(s) to be
+  deleted.
+- `player_idx::Integer` : Index of the player to delete action(s) for.
+
+# Returns
+
+- `::NormalFormGame` : `NormalFormGame` instance with the action(s) deleted as
+  specified.
+"""
+function delete_action(g::NormalFormGame{N},
+                       action::AbstractVector{<:PureAction},
+                       player_idx::Integer=1) where N
```

Comment:
Do not set a default for `player_idx` in this method for `NormalFormGame`. (Player 1 is not special in any way, is it?)